guides/v2.3/graphql/reference/customer.md

```diff
@@ -8,12 +8,17 @@ The `Customer` endpoint contains information about a customer account accessible
 To return or modify information about a customer, Magento recommends you use customer tokens in the header of your GraphQL calls. However, you also can use [session authentication]({{ page.baseurl }}/get-started/authentication/gs-authentication-session.html).
 
 ## Queries
-Use queries to read server-side data, such as a specific customer's address.
+
+The `customer` query returns information about the logged-in customer.
+
+The `isEmailAvailable` query checks whether the specified email has already been used to create a customer account. A value of `true` indicates the customer can use the email address to create an account.
```

Comment:
Question: Does a value of true indicate that the email has NOT already been used and can therefore be used to create an account?

Reply:
Correct.

Reply:
I know it's implied, but do you feel we should explicitly mention that the email has not already been used?